addons/mail/static/tests/discuss_app/discuss.test.js

```diff
@@ -245,7 +245,7 @@ test("Can use channel command /who", async () => {
     await start();
     await openDiscuss(channelId);
     await insertText(".o-mail-Composer-input", "/who");
-    await press("Enter");
+    await click("button[title='Send']");
```

Comment:
Why replace `press Enter` by `click Send` here?
People essentially send message in desktop with press Enter. Very rarely, barely anyone, uses the send button. This is more for mobile.

_(I even wonder if we shouldn't even remove it in desktop, very few chat competitors show it in desktop, but that's another debate... The benefit of removal is to give more horizontal real-estate which is precious in chat windows)_